src/main/java/frc/robot/Robot.java

```diff
@@ -120,6 +123,7 @@ public void teleopInit() {
   @Override
   public void teleopPeriodic() {
     Scheduler.getInstance().run();
+    DriveTrain.Manual(RobotMap.starboardMotor, RobotMap.portMotor, RobotMap.driverStick);
```

Comment:
`DriveTrain.Manual();` Does not exist, I think you meant to use `ManualDrive.Manual();`